index.css

```diff
@@ -0,0 +1,167 @@
+*,
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+ul,
+li {
+  margin: 0px;
+  padding: 0px;
+}
+body {
+  font-family: Arial, sans-serif;
+  background: #f5f5f5;
+}
+
+#todo-app {
+  max-width: 913px;
+  max-height: 517px;
+  margin: auto;
+
+  background: white;
+  padding: 20px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  
+}
+#todo-app h1 {
```

Comment:
I'm not quite sure why, but there seems to be a border around the entire app, you may have interpreted the figma grey as part of the app? It is not, the app should have had an all white background. 

-5

<img width="1512" alt="image" src="https://github.com/rumoravenue/rumor-react-training-a1/assets/62576642/4b3b5f90-267c-4353-9b3a-be7f3bb49d76">

Reply:
Actually if we see like this as plain UI it is a shadow effect in background   and can be implemented on the Parent div.
But in the Figma design I thought it is default UI of Figma , so that's why I haven't applied this shadow effect.


Thanks